packages/expo-cli/src/commands/client/selectDistributionCert.js

```diff
@@ -13,6 +13,14 @@ export default selectDistributionCert;
 async function selectDistributionCert(context, options = {}) {
   const certificates = context.username ? await chooseUnrevokedDistributionCert(context) : [];
   const choices = [...certificates];
+
+  // autoselect creds if we find valid ones
+  if (certificates.length > 0 && !options.disableAutoSelectExisting) {
+    const autoselectedCertificate = certificates[0];
```

Comment:
We take the first cert from the array here — how is this array sorted?

I believe we should prefer the one that shows as ”used in @username/home` (or whatever the slug off the client is).